### DISCUSSION TITLE
Does anyone knows how to display table across two columns?

### DISCUSSION THREAD

**Discussion by abcvav** [typst/typst#1603] [Q&A]
In the IEEE template that is officially provided, text is displayed with two clolumns:
```
show: columns.with(2, gutter: 12pt)
```

But if I want to display a table or a figure across two columns, anyone knows how to do that?
![image](https://github.com/typst/typst/assets/47599849/dcb457c4-0be1-43ad-a485-bb00ddfcbc26)

Like this:
![image](https://github.com/typst/typst/assets/47599849/3ea827fe-c389-4437-a634-0c73b9d67694)

**Comment by way-zer** [comment]
I found a workaround:
Use place in left column
```typst
#place(top,float: true)[
  #show: box.with(width: 200%) // use two column width
  #figure(caption: [Delay and LossRate in Time])[
  #rect(width: 100%, height: 100pt)
  ]
]
```
And place `#v(200pt)` in right column.

However, this method requires manual adjustment of the command position.
Look forward to #553 resolved.
